src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs

```diff
@@ -770,7 +770,7 @@ public Task<string> GetLastCommitShaAsync(string repoUri, string branch)
         /// <param name="owner">Owner of repo</param>
         /// <param name="repo">Repository name</param>
         /// <param name="branch">Branch to retrieve the latest sha for</param>
-        /// <returns>Latest sha.  Throws if no commits were found.</returns>
+        /// <returns>Latest sha.  Null if no commits were found.</returns>
```

Comment:
Are there any callers that were expecting a throw here?

Reply:
No. There are only 2 callers for this function: 1 is the GetLastCommitShaAsync(repoUri, branch) function, which is expected to return null if no commits were found (per the comment), and CreateBranchAsync, which doesn't catch, and appears to expect this function to return something to it, either a sha or null.

Reply:
Great. Thanks!